packages/Interface/src/DTK_Fortran_API.f90

```diff
@@ -111,6 +112,7 @@ subroutine swigc_DTK_set_function(farg1, farg2, farg3, farg4) &
    type(C_PTR), value :: farg4
   end subroutine
  end interface
+#endif
```

Comment:
Not sure what to do.  Anybody has experience with Doxygen for generating Fortran doc?